KubeArmor/.goreleaser.yaml

```diff
@@ -9,6 +9,20 @@ builds:
       - amd64
       - arm64
 
+signs:
+  - cmd: cosign
+    env:
+      - COSIGN_EXPERIMENTAL=1
```

Comment:
Thank you for working on this. 

minor nit: can be removed, this was required in cosign < 2.0.0 (not a blocker from my end.)